action.yml

```diff
@@ -134,23 +134,25 @@ runs:
           --volume "${PWD}/${{ inputs.output-dir }}:/${{ inputs.output-dir }}"
           --volume "${GITHUB_ACTION_PATH}:/github/action"
         install: |
+          export DEBIAN_FRONTEND=noninteractive
+          export TZ=UTC
           apt-get update -y
-          apt-get install -y software-properties-common
-          add-apt-repository -y ppa:deadsnakes/ppa
-          apt-get update -y
-          apt-get install --ignore-missing -y curl python${{ inputs.python-version }} python${{ inputs.python-version }}-dev pkg-config gcc g++ patchelf binutils zlib1g-dev ${{ inputs.additional-arm-packages }}
-          # Install pip for requested Python version using get-pip.py as ensurepip does not work here
-          curl -sS https://bootstrap.pypa.io/get-pip.py | python${{ inputs.python-version }}
-          python${{ inputs.python-version }} -m pip install --upgrade pip setuptools wheel
+          apt-get install -y curl git build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev pkg-config gcc g++ patchelf binutils ${{ inputs.additional-arm-packages }}
         run: |
           adduser --disabled-password --gecos "" builder
           chmod -R a+rwx /home/runner/work
           su builder <<'EOF'
+          curl -LsSf https://astral.sh/uv/install.sh | sh
+          source $HOME/.local/bin/env
+          # Install Python version using uv
+          uv python install ${{ inputs.python-version }}
+          # Install pip and upgrade tools
+          python -m pip install --upgrade pip setuptools wheel
```

Comment:
With uv you don't need pip. Use uv for installation and has a faster package resolution algorithm in comparison with pip. No need to do "pip install" anywhere.

Reply:
I think you will need `uv tool install setuptools wheel`.

And then you can invoke any tool with `uv run`.